src/SignalRServiceExtension/Constants.cs

```diff
@@ -7,6 +7,7 @@ internal static class Constants
     {
         public const string AzureSignalRConnectionStringName = "AzureSignalRConnectionString";
         public const string AzureSignalREndpoints = "Azure:SignalR:Endpoints";
+        public const string AzureSignalRHubProtocol = "Azure:SignalR:HubProtocol";
```

Comment:
if someone use standard2, then it doesn't work

Reply:
can this protocol config also apply to netstandard2.0?

Reply:
This protocol config is designed for netcore3.1